circuit_setup/inputs/mdl1/device_private_key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Can we generate the sample device and issuer private keys, rather than hardcode them?  For device bound JWTs we do, see https://github.com/microsoft/crescent-credentials/blob/3397e0052e04fbc57ff4768f6ff0ede087433111/circuit_setup/scripts/run_setup.sh#L91 

I see that there is a script in this PR to generate them that gets called as setup, so maybe these *.pem files should not be checked in.

Reply:
Yeah, they shouldn't be needed, the setup scripts should generate them. I'll delete them and make sure everything is still fine.

Reply:
Removed the checked-in files that are generated by the setup script